### PR TITLE
feat: linebreak fixes for Dartmoor intel on the Carlisle's

### PR DESCRIPTION
**Pull request by dribbleondo**
feat: linebreak fixes for Dartmoor intel on the Carlisle's

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
 	"$schema": "https://raw.githubusercontent.com/atampy25/simple-mod-framework/main/Mod%20Manager/src/lib/manifest-schema.json",
-	"version": "0.30.0",
+	"version": "0.12.0",
```
Comment (RDIL) [inline manifest.json:3]: This doesn’t seem right.

File: manifest.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 	"id": "GlacierOrganisation.CommunityPatches",
 	"name": "Community Patches",
 	"description": "A collection of fixes for Hitman: World of Assassination. This ranges from localisation line break fixes, inverted normals on clothing, and everything in-between. If it's broken in-game, this project aims to fix it!",
 	"authors": ["Notex, Minnow, Atampy26, Jojje, Dribbleondo, musicalmushr00m, VoodooHillbilly"],
 	"contentFolders": ["content/SmallFixes", "content/MovieSetLamp", "content/SapienzaChurch"],
 	"blobsFolders": ["blobs"],
 	"frameworkVersion": "2.33.9",
-	"updateCheck": "https://github.com/glacier-modding/H3-Community-Patches/releases/latest/download/updates.json",
```
Comment (RDIL) [inline manifest.json:11]: Same here.

**Review by RDIL — CHANGES_REQUESTED** [review]

**Comment by Ocean-Minnow** [comment]
As RDIL has pointed out above, this will most likely break automatic updates. The version number and update check URL have changed when they shouldn't have.

**Comment by dribbleondo** [comment]
Oh, lol, that's strange. Yeah, I'll fix that.

**Comment by Ocean-Minnow** [comment]
Okay, it looks like the github-actions bot has overwritten the fix again with its most recent `chore: mod update` commit. I'm not really sure what's happening here, but it may be best to make a new PR and see if that changes anything.

**Pull request merged**